test/library/metrics/query.go

```diff
@@ -38,26 +37,15 @@ func NewPrometheusClient(ctx context.Context, kclient kubernetes.Interface, rc r
 		return nil, fmt.Errorf("failed to get thanos-querier route: %w", err)
```

Comment:
Who uses this function? How are they going to behave after 24 hours, when the token expires?

Reply:
This is only used during e2e tests. I don't recall any tests lasting longer than 24hrs.